docs/guide.md

```diff
@@ -148,7 +148,7 @@ The same applies for the options of your currency input component. Use the `setO
 </template>
 
 <script>
-import { watch } from 'vue' // Vue 2: import { watch } from '@vue/composition-api'
+import { watch } from 'vue' // Vue 2.6 or earlier: import { watch } from '@vue/composition-api'
```

Comment:
Comment can be removed, since only Vue 2.7+ is supported:

```suggestion
import { watch } from 'vue'
```

Reply:
Thanks for review. I submit the change.